docs/Helm.md

```diff
@@ -0,0 +1,168 @@
+```
+
+2. Using the Helm CLI directly to set new values
+```
+helm install --set <KEY>=<VALUE> <RELEASE_NAME> splunk/splunk-operator -n <RELEASE_NAMESPACE>
+```
+
+If the release already exists, we can use ```helm upgrade``` to configure and upgrade the deployment using a file or the CLI directly as above.
+
+```
+helm upgrade -f new_values.yaml <RELEASE_NAME> splunk/splunk-operator -n <RELEASE_NAMESPACE>
+```
+
+Read more about configuring values [here](https://helm.sh/docs/intro/using_helm/).
+
+In the following example, we will install and upgrade the Splunk Operator.
+
+Specify the release name and namespace to install the Operator:
+
+```
+helm install splunk-operator-test splunk/splunk-operator -n splunk-operator
+```
+```
+NAME: splunk-operator-test
+LAST DEPLOYED: Tue Aug 23 12:47:57 2022
+NAMESPACE: splunk-operator
+STATUS: deployed
+REVISION: 1
+TEST SUITE: None
+```
+```
+NAME                                                  READY   STATUS    RESTARTS   AGE
+splunk-operator-controller-manager-545cccf79f-9xpll   2/2     Running   0          2m14s
+```
+By default, the Splunk Operator has cluster-wide access. Let's upgrade the ```splunk-operator-test``` release by revoking cluster-wide access:
+```
+helm upgrade --set splunkOperator.clusterWideAccess=false splunk-operator-test splunk/splunk-operator -n splunk-operator
+```
+```
+NAME: splunk-operator-test
+LAST DEPLOYED: Tue Aug 23 12:53:08 2022
+NAMESPACE: splunk-operator
+STATUS: deployed
+REVISION: 2
+TEST SUITE: None
+```
+Finally, let's terminate the Splunk Operator by uninstalling the ```splunk-operator-test``` release:
+```
+helm uninstall splunk-operator-test -n splunk-operator
+```
+```
+release "splunk-operator-test" uninstalled
+```
+
+## Splunk Enterprise deployments
+
+The Splunk Enterprise chart allows you to install and configure Splunk Enterprise custom resources. The ```splunk/splunk-enterprise``` chart has the ```splunk/splunk-operator``` chart as a dependency by default. To satisfy the dependencies please use the following command: 
+```
+helm dependency build splunk/splunk-enterprise
+```
+If the operator is already installed then you will need to disable the dependency:
+```
+helm install --set splunk-operator.enabled=false <RELEASE_NAME> splunk/splunk-enterprise -n <RELEASE_NAMESPACE>
+```
+Installing ```splunk/splunk-enterprise``` will deploy Splunk Enterprise customer resources according to your configuration, the following ```new_values.yaml``` file specifies override configurations to deploy a Cluster Manager, an Indexer Cluster and a Search Head Cluster.
+
+```
+clusterMaster:
+  enabled: true
+  name: cm-test
+
+indexerCluster:
+  enabled: true
+  name: idxc-test
+
+searchHeadCluster:
+  enabled: true
+  name: shc-test
+```
+The configurations above will override values in ```splunk/splunk-enterprise``` values file.  To see all configurable values contained in the ```values.yaml``` file:
```

Comment:
Nit: If we could somehow mention that by default the `values.yaml `without any override deploys a `standalone` that would be good. The current statement would be fine too if not possible.